docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -120,6 +120,74 @@
     "estimator.options.twirling.shots_per_randomization = 100"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e5dcab7b",
+   "metadata": {},
+   "source": [
+    "## Probabilistic Error Amplification (PEA)\n",
+    "\n",
+    "PEA is an aplification technique used with [Zero-noise extrapolation(ZNE)](#zne).  The main challenge in successfully implmenting ZNE is to have an accurate model for noise in the expectation value and to amplify the noise in a known way.  For utility-scale experiments, PEA is often the best choice because it can be applied to any circuit that can be run with native noise factor $(\\lambda - 1)$. \n",
+    "\n",
+    "Because PEA is a ZNE amplification technique, you also need to enable ZNE by setting `resilience.zne_mitigation = True`. Other `resilience.zne` options can be used in addition to set extrapolators, amplification levels, and so on. PEA requries a noise model, which is automatically generated when using primitives. \n",
+    "\n",
+    "When the Estimator primitive is used to call `noise-learner`, PEA can reuse the returned noise model. **LINK TO NOISE LEARNER DOCS** "
+   ]
```

Comment:
We are still not 100% sure if we'll enable reusing the noise learning data for follow-up experiments. If we do (and I will give you a head-up), you may want to add this after [this snippet](https://github.com/Qiskit/documentation/pull/1777/files#r1715951505):

When choosing to use PEA, the estimator automatically performs the noise reconstruction in preparation for the actual mitigation experiment. The results of this "noise learning" stage are reported in the metadata of the estimator's results. They can be re-used in subsequent experiments, potentially leading to significantly improved runtimes:

```
estimator = Estimator(mode=backend)
estimator.options.resilience.zne_mitigation = True 
estimator.options.resilience.zne.amplifier = "pea" 

# the first job performs the noise reconstruction in preparation to the actual estimation task
job1 = estimator.run(pubs, precision=0.01)

# the noise data is available in the results
noise_data = job1.result().metadata["resilience"]["layer_noise_model"]

# it can be re-used in subsequent estimation jobs to skip the noise-learning stage
estimator.options.resilience.layer_noise_model = noise_data
job2 = estimator.run(pubs, precision=0.01)
```
The noise learning stage can also be performed independently of the estimation task via the `NoiseLearner` class:
```
learner = NoiseLearner(backend)
learner_job = learner.run(pubs)  # `run` accepts pubs or a list of circuits
noise_data = job.result()

estimator = Estimator(mode=backend) 
estimator.options.resilience.zne_mitigation = True 
estimator.options.resilience.zne.amplifier = "pea"
estimator.options.resilience.layer_noise_model = noise_data
estimator_job = estimator.run(pubs, precision=0.01)